packages/wdio-sauce-service/README.md

```diff
@@ -2,9 +2,12 @@ WebdriverIO Sauce Service
 =========================
 
 > WebdriverIO service that provides a better integration into Sauce Labs. This service can be used for:
-> - the Sauce Labs virtual machine cloud (desktop web and em/simulators) and can update the job metadata ('name', 'passed', 'tags', 'public', 'build', 'custom-data') and runs Sauce Connect if desired.
+> - the Sauce Labs virtual machine cloud (desktop web and em/simulators) and can update the job metadata ('name'*, 'passed', 'tags', 'public', 'build', 'custom-data') and runs Sauce Connect if desired.
 > - the Sauce Labs Real Device cloud (iOS and Android) and can **ONLY** update the job status to passed / failed
 
+> \* By default the Sauce Service will update the 'name' of the job when the job starts. This will give you the option to update the name at any given point in time.
```

Comment:
```suggestion
> - By default the Sauce Service will update the 'name' of the job when the job starts. This will give you the option to update the name at any given point in time.
```